R/estimate_grouplevel.R

```diff
@@ -62,7 +62,10 @@ estimate_grouplevel <- function(model, type = "random", ...) {
   # Remove columns with only NaNs (as these are probably those of fixed effects)
   random[vapply(random, function(x) all(is.na(x)), TRUE)] <- NULL
 
-  # Correct for fixed effect
+  # Filter more columns
+  random <- random[, grepl("Group|Level|Parameter|Component|Median|Mean|MAP|Coefficient|CI|SE", names(random))]
+
```

Comment:
Shouldn't there be a closing parenthesis in the pattern?

Reply:
Oh, just saw there's no opening parenthesis as well. So this might work.

Reply:
don't think so?